plugins/modules/packaging/language/ansible_galaxy_install.py

```diff
@@ -226,6 +235,13 @@ def _get_ansible_galaxy_version(self):
 
     def __init_module__(self):
         self.ansible_version = self._get_ansible_galaxy_version()
+        if self.ansible_version < (2, 11):
```

Comment:
We should *not* issue this warning if `ack_min_ansiblecore211` is provided:
```suggestion
        if self.ansible_version < (2, 11) and not self.vars.ack_min_ansiblecore211:
```
I think we should also issue this warning i `ack_ansible29` is provided. What do you think?
```suggestion
        if (self.ansible_version < (2, 11) or self.vars.ack_ansible29) and not self.vars.ack_min_ansiblecore211:
```

Reply:
if `ack_ansible29` has all the same effects as `ack_min_ansiblecore211`, that defeats the purpose of `ack_min_ansiblecore211`. It will also hides the deprecation from users who already have set `ack_ansible29=true`.